identity/app/actions/AuthenticatedActions.scala

```diff
@@ -117,7 +117,10 @@ class AuthenticatedActions(
 
     def refine[A](request: AuthRequest[A]) =
       if (IdentityRedirectUsersWithLingeringV1ConsentsSwitch.isSwitchedOn && IdentityAllowAccessToGdprJourneyPageSwitch.isSwitchedOn)
-        decideConsentJourney(request)
+        if (!request.cookies.get("GU_CONSENTED").isDefined) // see comment in EditProfileController regarding GU_CONSENTED
```

Comment:
include this condition in the first one and combine the two `Future.successful(Right(request))`?